_common/qiskit/benchmark_runner.py

```diff
@@ -93,21 +93,20 @@
 
     args = parser.parse_args()
 
-# For Inserting the Noise model default into exec option as it is function call 
+    # For Inserting the Noise model default into exec option as it is function call 
     if args.noise_model != None :
         
         if args.noise_model == 'None':
            args.exec_options["noise_model"] = None 
         else:
             module,method = args.noise_model.split(".")
-            module = globals()[module]
+            module = importlib.import_module(f"custom.{module}")
             method = method.split("(")[0]
             custom_noise = getattr(module, method)
-            print(custom_noise)
             noise = custom_noise()
```

Comment:
You don't really need to define this as a variable, you can just set the next line directly equal to it. That is

```py
args.exec_options["noise_model"] = custom_noise()
```